blocks/testimonials/testimonials.js

```diff
@@ -0,0 +1,51 @@
+import {
+  div,
+  blockquote,
+  p,
+  cite,
+} from '../../scripts/dom-helpers.js';
+
+function parseTestimonial(text) {
+  const lastDashIndex = text.lastIndexOf(' - ');
```

Comment:
This is too much magic for authors.  If there's a dash in any part of the testimony this will break.  How about we add as second column that contains the author and date.

Reply:
My implementation already ignores dashes in the testimony. There's already an example of a dash in a testimonial

Reply:
I changed to the two column implementation, so no longer have to deal with this logic, thanks!